projects/control-service/projects/helm_charts/pipelines-control-service/values.yaml

```diff
@@ -320,6 +319,32 @@ security:
 
       ## [ Required if security.enabled = True ]
       jwtIssuerUrl: ""
+   kerberos:
+      ## These values contain configuration settings for the kerberos authentication provider in control-service.
+      ## The values are exposed as environment variables through the helm chart
+      ## which get picked up by our spring application (contorl-service)
+      ## on startup. More information about kerberos and spring can be found here:
+      ## https://docs.spring.io/spring-security-kerberos/docs/current/reference/htmlsingle/index.html
+      ## More information on kerberos authentication can be found here:
+      ## https://docs.oracle.com/cd/E23941_01/E26092/html/kerberos-auth.html
+      ## You only need to set the flag and property contents with correct values and kerberos authentication should work,
+      ## provided global security is enabled.
+      ##
+      ## The krb5.conf file that we use is the one specified in the credentials section and is required for kerberos
+      ## authentication to work. If kerberos authentication is enabled make sure the kerberosKrb5Conf property is set in
+      ## the credentials section as described in the documentation.
+      ##
+      ## Flag to determine whether kerberos authorization should be enabled/disabled
+      enabled: false
+      ## For more information on kerberos principal please review the official kerberos documentation:
+      ## https://web.mit.edu/kerberos/krb5-1.5/krb5-1.5.4/doc/krb5-user/What-is-a-Kerberos-Principal_003f.html
+      kerberosServicePrincipal: ""
```

Comment:
You should explicitly said the principal should have service type "HTTP" aka http/xxx - It's noted in the samples here - https://docs.spring.io/spring-security-kerberos/docs/current/reference/htmlsingle/index.html#samples-sec-server-win-auth but it's easy to miss.